datasets/dataset_loading_hub.py

```diff
@@ -212,7 +212,7 @@ def __init__(self):
         self.transform_validate = transforms.Compose(
             [
                 transforms.Resize(256),
-                transforms.CenterCrop(self.im_size.width),
+                transforms.CenterCrop(self.image_shape.width),
```

Comment:
When reviewing one can comment like so, next to code. Good stuff. Thanks for spotting this.